src/modules/user/components/user-contact-popup/user-contact-popup.tsx

```diff
@@ -8,33 +8,64 @@ import type { UserContactPopupProps } from './user-contact-popup.interface';
 import styles from './user-contact-popup.module.scss';
 
 export const UserContactPopup: FC<UserContactPopupProps> = ({ ...props }) => {
+  const handleSocialMediaClick = (url?: string | null) => {
+    if (url) {
+      window.open(url, '_blank');
+    }
+  };
+
+  const professional = props.professional;
+
   return (
     <Popover {...props} sideOffset={5} side='bottom'>
       <div className={styles.root}>
-        <Button
-          icon='phone'
-          className={clsx('focusable', styles.socialBtn, styles.phone)}
-          variant='unstyled'
-          rippleColor={styles.rippleColor}
-        />
-        <Button
-          icon='facebook'
-          className={clsx('focusable', styles.socialBtn, styles.facebook)}
-          variant='unstyled'
-          rippleColor={styles.rippleColor}
-        />
-        <Button
-          icon='instagram'
-          className={clsx('focusable', styles.socialBtn, styles.instagram)}
-          variant='unstyled'
-          rippleColor={styles.rippleColor}
-        />
-        <Button
-          icon='inbox'
-          className={clsx('focusable', styles.socialBtn, styles.mail)}
-          variant='unstyled'
-          rippleColor={styles.rippleColor}
-        />
+        {professional && professional.user.phone && (
+          <Button
+            onClick={() => {
+              if (professional.user.phone) {
+                window.open(`tel:${professional.user.phone}`);
+              }
```

Comment:
You don't need if here as you already made the check at the top. If it would be undefined it just won't render this button.